visualization/include/pcl/visualization/area_picking_event.h

```diff
@@ -50,12 +51,16 @@ namespace pcl
     class PCL_EXPORTS AreaPickingEvent
     {
       public:
-        AreaPickingEvent (int nb_points, const std::vector<int>& indices)
+        AreaPickingEvent (int nb_points, const std::map<std::string, Indices>& cloud_indices)
           : nb_points_ (nb_points)
-          , indices_ (indices)
+          , cloud_indices_ (cloud_indices)
```

Comment:
Considering that `std::move` is used to pass the indices int an `AreaPickingEvent` it could be interesting to take `cloud_indices` by value and to `std::move` it into `cloud_indices_`: that way there would be two moves of the `std::map` instead of a copy. This is a transformation that could be directly proposed by clang-tidy's [modernize-pass-by-value check](https://clang.llvm.org/extra/clang-tidy/checks/modernize-pass-by-value.html).

Reply:
Added std::move into contructor